streams/src/test/java/org/apache/kafka/streams/state/internals/NamedCacheTest.java

```diff
@@ -208,31 +206,6 @@ public void apply(final List<ThreadCache.DirtyEntry> dirty) {
         assertEquals(cache.flushes(), 1);
     }
 
-    @Test
```

Comment:
Should we update the tests instead of removing them? Or do you think it is already covered in other places?

Reply:
These tests are no longer applicable since they're testing the iterator's keyRange(), which is removed in this PR

Reply:
Yes, I was suggesting that maybe we should replace them with test cases covering `subMapIterator` instead; but after second thought I think it is not necessary since it is really just levering on J8's subMap function anyways, so we can save them.